scripts/require-sync/main.go

```diff
@@ -191,3 +199,34 @@ func opaClient(referential bool) (*opa.Client, error) {
 
 	return client, nil
 }
+
+func validateRequiresSyncDataContent(annotation string) (bool, error) {
+	allowedKeys := []string{"kinds", "groups", "versions"}
+
+	// Remove outer quotes
+	annotation = annotation[1 : len(annotation)-1]
+
+	// Validate JSON
+	if ok := json.Valid([]byte(annotation)); !ok {
+		return false, fmt.Errorf("Error validating JSON format")
+	}
+
+	// Unmarsall JSON
+	var contents []interface{}
+	if err := json.Unmarshal([]byte(annotation), &contents); err != nil {
+		return false, fmt.Errorf("Error validating JSON content")
+	}
+
+	// Validate keys
+	for _, value := range contents {
```

Comment:
nit: we should change the variable names to make intent clear and avoid confusion around shadowing.

Maybe: outer-loop-var => `requirement`, inner-loop-var => `equivalents`?

Reply:
Sure, those sound good to me.